ai-data/generative-apis/how-to/use-function-calling.mdx

```diff
@@ -146,6 +146,10 @@ response = client.chat.completions.create(
     The model automatically decides which functions to call. However, you can specify a particular function by using the `tool_choice` parameter. In the example above, you can replace `tool_choice=auto` with `tool_choice={"type": "function", "function": {"name": "get_flight_schedule"}}` to explicitly call the desired function.
 </Message>
 
+<Message type="important">
+    Some models shall be told they can use external functions in the system prompt. If you do not provide a system prompt when using tools, Scaleway will automatically add one that works best for that specific model.
```

Comment:
```suggestion
    Some models must be told they can use external functions in the system prompt. If you do not provide a system prompt when using tools, Scaleway will automatically add one that works best for that specific model.
```

Reply:
Even though it's preferential, I'd rather not use "shall" as it can be misinterpreted by french users. "Must" is more explicit and clear.